ci-operator/config/openshift/operator-framework-olm/openshift-operator-framework-olm-master.yaml

```diff
@@ -69,6 +73,22 @@ tests:
         requests:
           cpu: 100m
     workflow: ipi-aws
+- as: e2e-aws-console-olm
+  steps:
+    cluster_profile: aws
+    test:
+    - as: test
+      cli: latest
+      commands: ./test-prow-e2e.sh olmFull
```

Comment:
I need to double-check where that script is present as I see it used frequently throughout the release repository, but I'm not sure if `olmFull` makes some implications as to where that OLM source code lives.